hack/helm/victoria-logs-single.yaml

```diff
@@ -0,0 +1,2 @@
+podDisruptionBudget:
```

Comment:
should we remove helm folder?

Reply:
No, files in helm folder are used in CI for running testing.  Files in root of `hack/` are used only to run linter.